CMakeLists.txt

```diff
@@ -75,6 +75,8 @@ if(NOT DISABLE_INSTALL)
   install(TARGETS ${IPFS_API_LIBNAME} DESTINATION lib)
   install(FILES include/ipfs/client.h DESTINATION include/ipfs)
   install(FILES include/ipfs/http/transport.h DESTINATION include/ipfs/http)
+  install(FILES ${json_SOURCE_DIR}/include/nlohmann/json.hpp DESTINATION include/nlohmann)
+  install(FILES ${json_SOURCE_DIR}/include/nlohmann/json_fwd.hpp DESTINATION include/nlohmann)
```

Comment:
this really looks like a hack. 

But anyway, the include folder is actually added to the search path, using `include_directories`:

```cmake
include_directories("include")
```